checks/apps/gromacs/gromacs_containerized.py

```diff
@@ -0,0 +1,95 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+from hpctestlib.sciapps.gromacs.benchmarks import gromacs_check
+
+# This is to test single node, multi-gpu performance with AMD's HIP port of GROMACS
+# available at: https://www.amd.com/en/technologies/infinity-hub/gromacs
+# docker pull amdih/gromacs:2022.3.amd1_174
+
+@rfm.simple_test
+class lumi_gromacs_containerized(gromacs_check):
+    maintainers = ['mszpindler']
+    use_multithreading = True
+    valid_prog_environs = ['builtin']
+    container_platform = 'Singularity'
+    num_nodes = 1
+    num_tasks_per_node = 8
+    num_gpus_per_node = 8
+    num_cpus_per_task = 6
+    valid_systems = ['lumi:gpu']
+    exclusive_access = True
+    time_limit = '15m'
+
+    nb_impl = parameter(['gpu'])
+
+    gromacs_image = parameter([
+        'docker://amdih/gromacs:2022.3.amd1_174',
+    ])
+
+    benchmark_info = parameter([
+        ('StandardMD/benchMEM', -1.08735e+06, 0.001), 			# atoms 81,743
+    #    ('StandardMD/benchPEP', -1.43527e+08, 0.001), 			# atoms 12,495,503
+        ('HECBioSim/Crambin', -204107.0, 0.001),      			# atoms 19,605
+        ('HECBioSim/Glutamine-Binding-Protein', -724598.0, 0.001),	# atoms 61,153
+        ('HECBioSim/hEGFRDimer', -3.32892e+06, 0.001),			# atoms 465,399
+    #    ('HECBioSim/hEGFRDimerSmallerPL', -3.27080e+06, 0.001),		# atoms 465,399
+        ('HECBioSim/hEGFRDimerPair', -1.20733e+07, 0.001),		# atoms 1,403,182
+        ('HECBioSim/hEGFRtetramerPair', -2.09831e+07, 0.001)		# atoms 2,997,924
+    ], fmt=lambda x: x[0], loggable=True)
+    
+    executable = 'gmx mdrun'
+
+    reference = {
+        'lumi:gpu': {
+            'HECBioSim/Crambin': (320.0, None, None, 'ns/day'),
+            'HECBioSim/Glutamine-Binding-Protein': (115.503, None, None, 'ns/day'),  
+            'HECBioSim/hEGFRDimer': (18.938, None, None, 'ns/day'),
+            'HECBioSim/hEGFRDimerSmallerPL': (33.785, None, None, 'ns/day'),
+            'HECBioSim/hEGFRDimerPair': (7.759, None, None, 'ns/day'),
+            'StandardMD/benchMEM': (92.043, None, None, 'ns/day'),
+        }
+    }
```

Comment:
You won't set the tolerance values?

Reply:
I think for the performance check to work with the `gromacs_check` class, they need to be defined as 'perf: {...}'. I think as it's now, there performance check will be ignored.